package.json

```diff
@@ -352,20 +353,25 @@
                     "group": "images_2_general@1"
                 },
                 {
-                    "command": "vscode-docker.images.push",
```

Comment:
The diff makes it hard to tell but this is actually inserting `pull` above `push` and re-indexing the others.